lib/manager/npm/post-update/lerna.spec.ts

```diff
@@ -20,26 +20,26 @@ describe('generateLockFiles()', () => {
     env.getChildProcessEnv.mockReturnValue(envMock.basic);
   });
   it('returns if no lernaClient', async () => {
+    const res = await lernaHelper.generateLockFiles({}, 'some-dir', {}, {});
```

Comment:
If you feel that `undefined` should still be supported here I can make that work again, let me know what you think.